static/app/views/replays/details.tsx

```diff
@@ -106,9 +107,21 @@ function getProjectSlug(event: Event) {
   return event.projectSlug || event['project.name']; // seems janky
 }
 
+// TODO: investigate the `:fullscreen` CSS selector
```

Comment:
I would add a scope to your TODO, to make it easier to ~ignore~ grep for. e.g. ` TODO(replay)`